[TR] [CI] Compile and run time JRE mismatch for `/api/skema` route

## Problem

Running the web app from the container causes an error when opening `/api/skema` route. Below is the stack trace.

```
end-to-end-rest-skema-tr-1  | ! @835hl3cc9 - Internal server error, for (GET) [/api/skema] ->                                                                                                                                                                 
end-to-end-rest-skema-tr-1  |                                                                                                                                                                                                                                 
end-to-end-rest-skema-tr-1  | play.api.UnexpectedException: Unexpected exception[RuntimeException: java.lang.UnsupportedClassVersionError: controllers/routes has been compiled by a more recent version of the Java Runtime (class file version 55.0), this v
ersion of the Java Runtime only recognizes class file versions up to 52.0]                                                                                                                                                                                    
end-to-end-rest-skema-tr-1  |   at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:358)                                                                                                                           
end-to-end-rest-skema-tr-1  |   at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:264)                                                                                                                                            
end-to-end-rest-skema-tr-1  |   at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:436)                                                                                                                                           
end-to-end-rest-skema-tr-1  |   at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:428)                                                                                                                                           
end-to-end-rest-skema-tr-1  |   at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)                                                                                                                                                           
end-to-end-rest-skema-tr-1  |   at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)                                                                                                                                                   
end-to-end-rest-skema-tr-1  |   at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)                                                                                                                                                               
end-to-end-rest-skema-tr-1  |   at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:63)                                                                                                                                       
end-to-end-rest-skema-tr-1  |   at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:100)                                                                                                                                   
end-to-end-rest-skema-tr-1  |   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)                                                                                                                                                     
end-to-end-rest-skema-tr-1  | Caused by: java.lang.RuntimeException: java.lang.UnsupportedClassVersionError: controllers/routes has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime onl
y recognizes class file versions up to 52.0                                                                                                                                                                                                                   
end-to-end-rest-skema-tr-1  |   at play.api.mvc.ActionBuilder$$anon$9.apply(Action.scala:384)                                                                                                                                                                 
end-to-end-rest-skema-tr-1  |   at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)                                                                                                                                                                      
end-to-end-rest-skema-tr-1  |   at play.api.libs.streams.StrictAccumulator.$anonfun$mapFuture$4(Accumulator.scala:168)                                                                                                                                        
end-to-end-rest-skema-tr-1  |   at scala.util.Try$.apply(Try.scala:213)                                                                                                                                                                                       
end-to-end-rest-skema-tr-1  |   at play.api.libs.streams.StrictAccumulator.$anonfun$mapFuture$3(Accumulator.scala:168)                                                                                                                                        
end-to-end-rest-skema-tr-1  |   at scala.Function1.$anonfun$andThen$1(Function1.scala:57)                                                                                                                                                                     
end-to-end-rest-skema-tr-1  |   at scala.Function1.$anonfun$andThen$1(Function1.scala:57)                                                                                                                                                                     
end-to-end-rest-skema-tr-1  |   at scala.Function1.$anonfun$andThen$1(Function1.scala:57)                                                                                                                                                                     
end-to-end-rest-skema-tr-1  |   at play.api.libs.streams.StrictAccumulator.run(Accumulator.scala:199)                                                                                                                                                         
end-to-end-rest-skema-tr-1  |   at play.core.server.AkkaHttpServer.$anonfun$runAction$4(AkkaHttpServer.scala:423)
end-to-end-rest-skema-tr-1  | Caused by: java.lang.UnsupportedClassVersionError: controllers/routes has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
end-to-end-rest-skema-tr-1  |   at java.lang.ClassLoader.defineClass1(Native Method)
end-to-end-rest-skema-tr-1  |   at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
end-to-end-rest-skema-tr-1  |   at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
end-to-end-rest-skema-tr-1  |   at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
end-to-end-rest-skema-tr-1  |   at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
end-to-end-rest-skema-tr-1  |   at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
end-to-end-rest-skema-tr-1  |   at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
end-to-end-rest-skema-tr-1  |   at java.security.AccessController.doPrivileged(Native Method)
end-to-end-rest-skema-tr-1  |   at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
end-to-end-rest-skema-tr-1  |   at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
```

It is a strange error because it doesn't affect other endpoints, such as those used for the unit tests.

--- a/skema/text_reading/scala/webapp/docker.sbt
+++ b/skema/text_reading/scala/webapp/docker.sbt
@@ -11,8 +11,7 @@ val tag = "1.0.0"
 
 
 Docker / defaultLinuxInstallLocation := appDir
-// TODO: can we run this with 11 (i.e., eclipse-temurin:11-jre-focal)?
-Docker / dockerBaseImage := "eclipse-temurin:8-jre-focal"
+Docker / dockerBaseImage := "eclipse-temurin:11-jre-focal"
 Docker / daemonUser := "nobody"
 Docker / dockerExposedPorts := List(port)
 Docker / maintainer := "Keith Alcock <docker@keithalcock.com>"

